### PR TITLE
feat(send to): Show a warning if sent package(s) contains published i…

### PR DESCRIPTION
**Pull request by ioanpocol**
…tems (SDFID-372)

### CODE REVIEW

File: scripts/apps/authoring/authoring/directives/SendItem.js [hunk 1]
```diff
@@ -540,6 +552,28 @@ export function SendItem($q, api, desks, notify, authoringWorkspace,
                 return $q.when(item);
             }
 
+            const confirmMovePackage = () =>
+                modal.confirm(
+                    gettext('On current package there are already published items that can\'t be moved.'),
```
Comment (tomaskikutis) [inline scripts/apps/authoring/authoring/directives/SendItem.js]: Grammar fix: The package contains published items which can\'t be moved.

File: spec/fetch_spec.js [hunk 2]
```diff
@@ -89,6 +89,7 @@ describe('fetch', () => {
 
         workspace.openIngest();
         content.actionOnItem('Fetch To', 0);
+        content.actionOnItem('Fetch To', 0);
```
Comment (tomaskikutis) [inline spec/fetch_spec.js]: Why perform the same action multiple times?

File: scripts/apps/authoring/authoring/directives/SendItem.js (hunk 1 above)
Comment (ioanpocol) [reply scripts/apps/authoring/authoring/directives/SendItem.js]: Done

Comment (tomaskikutis) [reply scripts/apps/authoring/authoring/directives/SendItem.js]: Thanks. Can you address the remaining comment? :)

File: spec/fetch_spec.js (hunk 2 above)
Comment (ioanpocol) [reply spec/fetch_spec.js]: Fixed

**Pull request merged**